packages/kbn-guided-onboarding/README.md

```diff
@@ -1,3 +1,7 @@
 # @kbn/guided-onboarding
 
-Empty package generated by @kbn/generate
+To run in development you will need the following settings in your kibana.dev.yml:
+
+xpack.cloud.id: 'testID'
+elasticsearch.hosts: ["http://localhost:9200"]
```

Comment:
We probably don't need to mention this line, it is typical for all environments.

Reply:
removed!